docs/sdk/01-examples/02-multisig-client/12-pin-metadata.md

```diff
@@ -26,26 +26,26 @@ const metadata: ProposalMetadata = {
   resources: [
     {
       name: "Discord",
-      url: "https://discord.com/..."
+      url: "https://discord.com/...",
     },
     {
       name: "Website",
-      url: "https://website..."
+      url: "https://website...",
     },
   ],
   media: {
     logo: "https://...",
-    header: "https://..."
+    header: "https://...",
   },
 };
 
 const metadataUri: string = await multisigClient.methods.pinMetadata(metadata);
-console.log({ metadataUri });
+console.log(metadataUri);
```

Comment:
Let's use the stanrdard `({ metadataUri })` so the console log is explicit about what it is returning.